### ISSUE TITLE
[ENHANCE] time jump

### ISSUE THREAD

**Issue by Soyaro666** [DarkholmeTenk/TardisModIssues#184]
I know it's not actyally possible to jump in time like it is in the show. but it is possible to change daytime. maybe there is an "unknown problem" interferring if there are players in that dimension and of course it doesn't do anything in worlds without time (nether or thr end, for example) and it should have a config option to turn it on/off (default should be off I think).
just thought about the tardis actually being a time machine and had this in mind

of course I coult simply go to bed but what if I want to have a specific daytime or a specific moon phase? that are variables I would want to see in such a function too.
Maybe you could add 2 wheels (daytime and moonphase) and one of the pullable button-things to de/activate them

**Comment by Jwtc2000** [comment]
I could see this as a Single Player Only Feature

**Comment by Soyaro666** [comment]
not really. it could work in multiplayer too if no players are in destination dim.
actually it would work similar to go to bed. you can go to bed if other players are in other dimensions, right?

**Comment by wrothmonk** [comment]
@Soyaro666 I can confirm that if all other players are in another dimension one (or multiple players) can go to bed and the time will change. I've regularly exploited that with a friend on our private server.

**Comment by FoxPotato** [comment]
tbh, I also see this as a single player only function, I mean. What dimensions would this be usefull in, if this mod was on a server and you'd need a special moon phase. you'd have to ask people to leave the overworld just because you want to change the time.
So even if it's added in smp, it wouldn't be all that useful unless you're playing with a friend.

**Comment by Soyaro666** [comment]
I'm playing with 4 friends on a server we pay together. 1 to 2 of them would live beside me in my tardis, one would live in an own dimension anyways. and a full moon-event is interesting for everybody I think. At least I could ask the last one to visit me in my Tardis to change time.
Beside that it would work in most Mystcraft worlds and multiple other mods add worlds with time cycle too I guess.
And if it works on the sleep mechanism it could have an option to work like morpheus mod. that adds a voting system, if half or more players are sleeping the time changes. for my suggestion this could be done like "if in the dimension are equal or less players than in my tardis, my tardis can change time". an option, maybe only available if morpheus is installed...
